docs/docs/docs/developer-resources/testing.md

```
@@ -1,10 +1,250 @@
+This section covers important tests to validate the operation of the API.
 
-Coming soon.
+## Accessing the API
+
+These are some important URLs for coding and troubleshooting:
+
+### Validation
+
+1. By default talawa-api runs on port 4000. It is available on the following endpoint:
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)